.github/workflows/deploy-cloudrun-it.yml

```diff
@@ -224,7 +224,7 @@ jobs:
         SERVICE: ${{ steps.service.outputs.service }}
         PARAMS: "{\"cpu\": 2,\"memory\": \"1Gi\", \"containerConcurrency\": 20}"
         ANNOTATIONS: "{\"run.googleapis.com/cloudsql-instances\": \"test-project:us-central1:my-test-instance\"}"
-        LABELS: "{\"test_label\": \"test_value\"}"
+        LABELS: '{"managed-by":"github-actions", "commit-sha":"${{ github.sha }}"}'
```

Comment:
The reason this test was working was because we weren't setting labels in deploy_2. If we set any labels in deploy_2, then the test_label:test_value would be overwritten as defined by gcloud.

Reply:
Regardless of deployment method I would expect our default labels to exist unless otherwise overridden. Might be confusing if is use the YAML and don't see those default values?

Reply:
Discussed over chat. We don't currently parse the YAML and this matches the behavior of already setting labels via YAML (they are ignored).